README.md

```diff
@@ -139,6 +139,23 @@ factory.define('user', User, {foo: 'bar'}, {
 });
 ```
 
+### Extending Factories
+
+You can extend a factory using `#extend`:
+
+```js
+factory.define('user', User, { username: 'Bob', expired: false });
+factory.extend('expiredUser', 'user', { expired: true });
```

Comment:
Looks like this example is wrong because according to source code the first and second args should be switched. So, the correct version is:

```js
factory.extend('user', 'expiredUser', { expired: true })
```

Reply:
The syntax feels a bit clunky, anyway to overload `define`

`define('expiredAdminUser', 'admin', 'user', User`
`define('expiredAdminUser, { model: User, extends: ['admin', 'user'] }, myInitializer...`

Reply:
Yes I did consider that, here's the alternative:

```js
factory.define('expiredUser', User, {
  expired: true
}, {
  extends: 'user'
});
// is equivalent to
factory.extend('user', 'expiredUser', { expired: true });
```

I think the `#extend` form is preferable.